docs/content/installation.md

```diff
@@ -266,24 +267,24 @@ This image is compressed with gzip to save space. In order to properly manage it
 
 ```bash
 backupobject add \
-  --uuid bo-lucid5d \
-  --display-name "Ubuntu 10.04 (Lucid Lynx) root partition" \
+  --uuid bo-ubuntu5d \
```

Comment:
The 5d was part of an old naming scheme used in development. There's no reason to keep it.